src/Microsoft.DotNet.Interactive.CSharpProject/BuildCacheFileUtilities.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Microsoft.DotNet.Interactive.CSharpProject.Packaging;
+using Microsoft.DotNet.Interactive.Utility;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Interactive.CSharpProject;
+
+public static class BuildCacheFileUtilities
+{
+    internal static string cacheFilenameSuffix = ".interactive.workspaceData.cache";
+    internal static string DirectoryBuildTargetFilename = "Directory.Build.Targets";
+
+    internal static string DirectoryBuildTargetsContent = $@"
```

Comment:
A raw string rather than a verbatim string would be nice here.

Reply:
done. In the latest commit